activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Add `Cache#fetch_with_deferred_update` to allow out-of-band updates of cache values.
+
+    ```ruby
+    Rails.cache.fetch_with_deferred_update(cache_key, 5.minutes) do
```

Comment:
what do you think about making this a keyword argument, so it's more explicit the purpose of it?

```suggestion
    Rails.cache.fetch_with_deferred_update(cache_key, refresh_grace_period: 5.minutes) do
```

I had to look at it a bit to understand, but I think the correct way to use the API is to have that value be roughly how long the block will take to complete (so it's the grace period during which the cache is being rebuilt). Is that right?

Reply:
I went with `:race_condition_ttl` as it is already an existing option for `fetch` :+1: